packages/performance/src/services/remote_config_service.test.ts

```diff
@@ -214,18 +226,145 @@ describe('Performance Monitoring > remote_config_service', () => {
     it('uses secondary configs if the response does not have any fields', async () => {
       // Expired local config.
       const EXPIRY_LOCAL_STORAGE_VALUE = '1556524895320';
-      const STRINGIFIED_PARTIAL_CONFIG = '{"state":"NO TEMPLATE"}';
+      const STRINGIFIED_PARTIAL_CONFIG = '{"state":"NO_TEMPLATE"}';
 
       setup(
         {
           expiry: EXPIRY_LOCAL_STORAGE_VALUE,
-          config: 'not a valid config and should not be used'
+          config: NOT_VALID_CONFIG
         },
         { reject: false, value: new Response(STRINGIFIED_PARTIAL_CONFIG) }
       );
       await getConfig(IID);
 
       expect(SettingsService.getInstance().loggingEnabled).to.be.true;
     });
+
+    it('marks event destination to cc if there is no template', async () => {
+      setup(
+        {
+          // Expired local config.
+          expiry: '1556524895320',
+          config: NOT_VALID_CONFIG
+        },
+        { reject: false, value: new Response('{"state":"NO_TEMPLATE"}') }
+      );
+      await getConfig(IID);
+
+      // If no template, will send to cc.
+      expect(SettingsService.getInstance().shouldSendToFl).to.be.false;
```

Comment:
`shouldSendToFl` -> `false` doesn't mean it sent to `CC`. Why not check if it actually sent to `Fl`?

Reply:
`remote config service` is working as a retriever of config, but it doesn't execute on event dispatch. Therefore we can only check the value in `Setting service`, `Setting service` serves as a config resolver in Web SDK.